src/source-control/GitSourceControl.ts

```diff
@@ -19,15 +31,18 @@ function isSttackBranch(branch: BranchName): boolean {
   return branch.startsWith("sttack-");
```

Comment:
Is this a merge mistake? 2 return statements in one function

Reply:
Yes, probably, lol, lemme fix this

Reply:
Lol, wait no, you collapsed the lines in between. I think, right?

Reply:
Oh shit sorry, that was stupid :sweat: